### DISCUSSION TITLE
Can OpenMPF detect vechicles?

### DISCUSSION THREAD

**Discussion by jrobble** [openmpf/openmpf#1539] [Q&A] [answered]
Like cars, trucks, and bikes?

**Comment by brosenberg42** [comment, marked as answer]
The [OcvYoloDetction component](https://github.com/openmpf/openmpf-components/blob/master/cpp/OcvYoloDetection) detects various objects including some vehicles. The list of objects it detects is [here](https://github.com/openmpf/openmpf-components/blob/master/cpp/OcvYoloDetection/plugin-files/models/coco.names). You can provide a [whitelist](https://github.com/openmpf/openmpf-components/blob/master/cpp/OcvYoloDetection/plugin-files/descriptor/descriptor.json#L76) so that the component only outputs the classes you are interested in.
